app/package.json

```diff
@@ -23,7 +23,8 @@
     "migrate:latest": "./node_modules/.bin/knex migrate:latest",
     "migrate:rollback": "./node_modules/.bin/knex migrate:rollback",
     "migrate:up": "./node_modules/.bin/knex migrate:up",
-    "migrate:down": "./node_modules/.bin/knex migrate:down"
+    "migrate:down": "./node_modules/.bin/knex migrate:down",
+    "posttest": "node ./lcov-fix.js"
```

Comment:
Optional: May not be a bad idea to have posttest next to the pretest function as it'll be easier to find.